### PR TITLE
make default appfile as vela.yaml

### LINKED ISSUE #432: [BUG] vela up trying find vela.yml instead of vela.yaml
Version: latest master branch build.

```console
...
    up              	Apply an appfile, by default vela.yml
```

Not sure why we assume this as `.yaml` is the widely adopted preference. See: https://yaml.org/faq.html

### PR DESCRIPTION
**Pull request by hongchaodeng**
fix #432

### CODE REVIEW

File: pkg/appfile/appfile.go [hunk 1]
```diff
@@ -17,7 +17,7 @@ var (
 	ErrImageNotDefined = errors.New("image not defined")
 )
 
-const DefaultAppfilePath = "./vela.yml"
+const DefaultAppfilePath = "./vela.yaml"
```
Comment (wonderflow) [inline pkg/appfile/appfile.go:20]: let's support both of them

Comment (wonderflow) [reply pkg/appfile/appfile.go:20]: check vela.yaml first and then vela.yml

Comment (resouer) [reply pkg/appfile/appfile.go:20]: Let's use `.yaml` only. It's rare and weird for a tool support both.

Comment (wonderflow) [reply pkg/appfile/appfile.go:20]: I'm OK with it。

**Review by wonderflow — APPROVED** [review]

**Pull request merged**